Frends.Community.Apache/Frends.Community.Apache.csproj

```diff
@@ -20,9 +20,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="CsvHelper" Version="27.1.1" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="Parquet.Net" Version="3.9.0" />
+    <PackageReference Include="CsvHelper" Version="30.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
```

Comment:
Please downgrade to 12.0.1, since Frends is using that version. Later version might work or not.